src/DurableTask.Core/TaskHubClient.cs

```
@@ -43,6 +43,17 @@ public TaskHubClient(IOrchestrationServiceClient serviceClient)
             this.defaultConverter = new JsonDataConverter();
         }
 
+        /// <summary>
+        ///     Create a new TaskHubClient with the given OrchestrationServiceClient and JsonDataConverter.
+        /// </summary>
+        /// <param name="serviceClient">Object implementing the <see cref="IOrchestrationServiceClient"/> interface </param>
+        /// <param name="dataConverter">The <see cref="JsonDataConverter"/> to use for message serialization.</param>
```

nit: Maybe change this to "The default <see cref="JsonDataConverter"/> to use for message serialization.", just so it is more clear that this is only the default, and not used for all message serialization.

---

I think the name `defaultConverter` in the existing DTFx code is actually misleading. It's not actually the "default". Rather, it's the _only_ converter used for serialization. The naming is probably some artifact of an aspiration they once had to make it extensible. In that light, I think Amanda's description is actually more correct as written.